doc/radosgw/keystone.rst

```diff
@@ -145,3 +148,11 @@ configurable ``rgw keystone verify ssl`` to false.
 
 
 .. _Openstack keystone documentation: http://docs.openstack.org/developer/keystone/configuringservices.html#setting-up-projects-users-and-roles
+
+
+Keystone integration with the S3 API
```

Comment:
@Miouge1 (here)

Reply:
My bad, looks good.

Reply:
... except I of course botched the cross-reference. :( Pushing up a fix.